### ISSUE TITLE
Add time manually (without any particular start/end)

### ISSUE THREAD

**Issue by jose-ba** [danniehansen/activity-timer#123]
Hello! First of all thank you for your work, the plugin works great.

I think that it would be nice to have the option to add a determinate amount of time to the card without the need to select a start and end time. For example, if you want to add 2 hours and 45 minutes, now you would have to add a start date and time, and a end date and time 2 hours and 45 minutes later. My idea is that there could be a new option, where there would be a text input where you could type 2:45, then click a button, and it would be added to the card.

Thank you

**Comment by danniehansen** [comment]
Hi @jose-ba,

I would definitely be supportive of such a feature. However, Activity timer heavily relies upon actual time intervals with timestamps for reporting. What i was thinking that we could do, is have it add the time tracking at the point of adding and then taking it the matching time back in the past. So if i report 3 hours on 4PM, then the time range would be 1PM - 4PM.

**Comment by jose-ba** [comment]
Yes, I understand, it would be a good solution. It would be a nice way to for example, add time spent on a meeting, or outside the office, etc.

Thank you!

**Comment by danniehansen** [comment]
@jose-ba This has been resolved with the recent release of the Activity timer successor [Limita](https://limita.org/)

Feel free to give it a look. If you're interested in trying it out, shoot me a message on [support@limita.org](mailto:support@limita.org) and i can drop you a discount code for a few months. It has a Activity timer migration tool that automatically carry over historical data.

I will close this issue for now 👍
